controls/ccn_rhel9.yml

```diff
@@ -0,0 +1,824 @@
+---
+policy: CCN-STIC-610A22
+title: Security Profile Application Guide for Red Hat Enterprise Linux 9
+id: ccn_rhel9
+version: '2022-10'
+source: https://www.ccn-cert.cni.es/pdf/guias/series-ccn-stic/guias-de-acceso-publico-ccn-stic/6768-ccn-stic-610a22-perfilado-de-seguridad-red-hat-enterprise-linux-9-0/file.html
+levels:
+  - id: basic
+  - id: intermediate
+    inherits_from:
+      - basic
+  - id: advanced
+    inherits_from:
+      - intermediate
+
+controls:
+  - id: reload_dconf_db
+    title: Reload Dconf database
```

Comment:
Title should be Title case per the [style guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#controls).